packages/@o3r/design/README.md

```diff
@@ -33,25 +33,27 @@ Otter Design module provides a set of code generators based on [Angular schemati
-| **templateFile**            | *null*         | Path to a template file (or a list of template files) to apply as default configuration to a Design Token extension.                 |
-| **prefix**                  | *null*         | Prefix to append to generated variables.                                                                                             |
-| **prefixPrivate**           | *null*         | Prefix to append to generated private variables.                                                                                     |
-| **watch**                   | false          | Enable Watch mode.                                                                                                                   |
+| Options                     | Default Value      | Description                                                                                                                                  |
+| --------------------------- | ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| **language**                | `'css'` *Required* | Default styling language to generate to the given Design Token. *(`css`, `scss` and `sass` are available)*                                   |
+| **designTokenFilePatterns** | `[]` *Required*    | Path patterns to the Design Token JSON files. <br /> Files in dependencies are supported and resolved with Node Resolver.                    |
+| **variableType**            | *null*             | Type of the variables to generate for a Design Token.                                                                                        |
+| **output**                  | *null*             | Output file where the CSS will be generated. <br /> The path specified in `o3rTargetFile` will be ignored if this option is specified        |
+| **defaultStyleFile**        | `'src/theme.scss'` | File path to generate the variable if not determined by the specifications                                                                   |
+| **metadataOutput**          | *null*             | Path to generate the metadata for the CMS. <br /> The metadata will be generated only if the file path is specified.                         |
+| **metadataIgnorePrivate**   | false              | Ignore private variables in the metadata generation.                                                                                         |
+| **rootPath**                | *null*             | Root path of files where the CSS will be generated.                                                                                          |
+| **failOnDuplicate**         | false              | Determine if the process should stop in case of Token duplication.                                                                           |
```

Comment:
```suggestion
| **failOnDuplicate**         | false              | Determine if the process should stop in case of token duplication.                                                                           |
```

Reply:
I think `Design Token`, `Token` and `Metadata` should remain with upper case.
Design Token is a standard name, same for Metadata that stands for the EAM Metadata protocol.
(We did already that in the current code)